<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserPage</class>
 <widget class="QWidget" name="UserPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>480</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item alignment="Qt::AlignmentFlag::AlignLeft">
    <widget class="QPushButton" name="btn_logout">
     <property name="text">
      <string>退出登录</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="headerFrame">
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <layout class="QHBoxLayout" name="headerLayout">
      <item>
       <widget class="QLabel" name="label_status">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>加载中…</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_refresh">
        <property name="text">
         <string>刷新</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="actionFrame">
     <layout class="QHBoxLayout" name="actionLayout">
      <item>
       <widget class="QPushButton" name="btn_park">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>停车</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_leave">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>离开</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="actionSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="group_reservation">
     <property name="title">
      <string>预订信息</string>
     </property>
     <layout class="QVBoxLayout" name="reservationLayout">
      <item>
       <widget class="QLabel" name="label_reservation">
        <property name="text">
         <string>正在加载预订信息…</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_reserve">
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>预订</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="group_order">
     <property name="title">
      <string>订单信息</string>
     </property>
     <layout class="QVBoxLayout" name="orderLayout">
      <item>
       <widget class="QLabel" name="label_order">
        <property name="text">
         <string>正在加载订单信息…</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="orderButtonLayout">
        <item>
         <widget class="QPushButton" name="btn_view_history">
          <property name="text">
           <string>查看支付信息和历史停车信息</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="orderButtonSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
